consumer-a/tests/getProducts.spec.ts

```diff
@@ -1,61 +1,26 @@
-import { products as mockedProducts } from "../fixtures/getProducts/200";
-import { Pact, Matchers } from "@pact-foundation/pact";
-import { resolve } from "path";
-import { getProducts } from "../src";
-
-const {like} = Matchers;
-
-const mockedProvider = new Pact({
-    consumer: "e2e Consumer A",
-    provider: "e2e Provider",
-    // port: 9292,
-    log: resolve(process.cwd(), "logs", "mockserver-integration.log"),
-    dir: resolve(process.cwd(), "pacts"),
-    logLevel: "warn",
-    spec: 2,
-})
-
-beforeAll(() => mockedProvider.setup().then(options => {
-    console.log(`http://localhost:${options.port}`);
-    process.env.API_HOST = `http://localhost:${options.port}`;
-}));
-
-afterAll(() => mockedProvider.finalize());
-
-afterEach(() => mockedProvider.verify());
-
-describe("consumer-a/getProducts()", () => {
+import { pactWith } from 'jest-pact';
+import { API, api } from "../src";
+import { productsRequest, withListOfProductsRequest } from "provider/src/endpoints";
 
+pactWith({ consumer: 'e2e Consumer A', provider: 'e2e Provider' }, provider => {
```

Comment:
feedback:

this looks nice, the abstraction with jest-pact makes sense, will adapt. Thanks